avocado/core/xunit.py

```diff
@@ -199,9 +199,7 @@ def end_test(self, state):
             self.xml.add_skip(state)
         elif status == 'FAIL':
             self.xml.add_failure(state)
-        elif status == 'ERROR':
-            self.xml.add_error(state)
-        elif status == 'INTERRUPTED':
+        else:   # ERROR, INTERRUPTED, ...
```

Comment:
I agree that setting this as `ERROR` would be better at spotting "driver" problems.  What I mean is, in theory, our result "drivers" should complain out loud when given unknown test status.  Since they lack that kind of communication back with the test runner, for now, setting `ERROR` is more adequate IMHO.